frontend/scripts/react-components/shared/search.component.js

```diff
@@ -0,0 +1,221 @@
+import { h, Component } from 'preact';
+import 'styles/components/shared/search.scss';
+import 'styles/components/shared/autocomplete.scss';
+import Downshift from 'downshift/preact';
+import { findAll } from 'highlight-words-core';
+import cx from 'classnames';
+
+import { PROFILE_PAGES_WHITELIST } from 'constants';
+
+import NodeTitleGroup from 'containers/shared/node-title-group-react.container';
+
+export default class Search extends Component {
+
+  static getNodeId(selectedItem) {
```

Comment:
didn't even know JS had static methods. Anyway, is there any reason for this particular method to be static? Looks like it should be "private", if anything

Reply:
This is an utility method, it doesn't depend on the instance, and its a pure function. I could move it outside of the class as a helper function, however I think that it makes more sense on leaving it inside because its totally dependent of the class implementation. There's no private methods in js, only static (class methods) and public.

Reply:
:+1: